### PR TITLE
feat(angular): update jest-preset-angular to 8.3.2

### PR DESCRIPTION
**Pull request by mehrad-rafigh**
## Current Behavior
jest-preset-angular is at v8.3.1

## Expected Behavior
jest-preset-angular is at v8.3.2

### CODE REVIEW

File: packages/angular/src/migrations/update-11-1-2/update-11-1-2.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { chain } from '@angular-devkit/schematics';
```
Comment (vsavkin) [inline packages/angular/src/migrations/update-11-1-2/update-11-1-2.ts:1]: You can remove this migration file. It's not used.

Comment (mehrad-rafigh) [reply packages/angular/src/migrations/update-11-1-2/update-11-1-2.ts:1]: The chain is used in the schematic @vsavkin. It used to update the packages and formatting files

Comment (vsavkin) [reply packages/angular/src/migrations/update-11-1-2/update-11-1-2.ts:1]: I mean we don't need the whole file.

Packages are updated automatically based on the versions specified in migrations.json

Comment (mehrad-rafigh) [reply packages/angular/src/migrations/update-11-1-2/update-11-1-2.ts:1]: Thats good to know. I will update the PR

File: packages/angular/migrations.json [hunk 2]
```diff
@@ -279,6 +279,15 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "11.1.5": {
```
Comment (mehrad-rafigh) [inline packages/angular/migrations.json:283]: 11.1.5 is already released. This should be updated to 11.1.6

**Comment by vsavkin** [comment]
Thank you!

**Pull request merged**